src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -42,6 +49,16 @@
 
 namespace Raven.Server.Documents.TcpHandlers
 {
+    public class SubscriptionOperationScope
+    {
+        // a Connection is composed of: PendingConnection + ActualConnection
+        // a Batch is composed of: SendDocuments + Ack
+        public const string PendingConnection = "PendingConnection";
```

Comment:
maybe rename field names to:
ConnectionPending
ConnectionActive
BatchSendingDocuments
BatchWaitingForAcknowlege 

and then you can remove comments ;-)

Reply:
done